.travis.yml

```diff
@@ -10,6 +10,10 @@ cache: bundler
 # Passes arguments to bundle install (http://gembundler.com/man/bundle-install.1.html)
 # bundler_args:
 
+before_install:
+  - gem update --system
```

Comment:
I'm not sure I understand this part. Based on the ref'd ticket, can we not just wait for this to be fixed in travis so as to avoid a workaround?